### DISCUSSION TITLE
I have no access to Copilot

### DISCUSSION THREAD

**Discussion by treasunew** [community/community#114022] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/104483693/b56889ba-8605-496a-a4a1-5c9cff3599db)
I have no idea on this, I am a verified student having trouble accessing copilot in pycharm

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
